cog/core/sql_abstract.py

```
@@ -0,0 +1,206 @@
+        return UserRecord(
+            uid=uid,
+            DCname=None,
+            DCMail=None,
+            githubName=None,
+            githubMail=None,
+            loveuwu=False,
+            point=0,
+            ticket=1,
+            charge_combo=0,
+            next_lottery=0,
+            last_charge=datetime(1970, 1, 1, 0, 0, 0),
```

Ruff (0.8.2)

`datetime.datetime()` called without a `tzinfo` argument

(DTZ001)